packages/ui/package.json

```diff
@@ -25,9 +26,9 @@
   },
   "exports": {
     ".": {
-      "types": "./dist-web/web.d.ts",
-      "import": "./dist-web/web.js",
-      "require": "./dist-web/web.js"
+      "types": "./dist-all/web.d.ts",
```

Comment:
Apologies for the faffing here but when I changed this in https://github.com/leather-io/mono/pull/228 I missed a step / encountered an un-expected issue. 

The `build:all` step was also creating `panda.` files in `dist-all` and I'm still not quite sure how it works so I decided to rollback and finish migrating the library, then optimise it. 

![Screenshot 2024-06-20 at 14 20 16](https://github.com/leather-io/mono/assets/2938440/def90457-9d87-4cf2-9825-882762a2d33a)